explainer.md

```diff
@@ -117,45 +119,20 @@ mediaSource.onsourceopen = (e) => {
   // App-specific request
   const mediaRequest = Uint8Array.from([1, 2, 3, 4]);
   const requestStream = await transport.createSendStream();
-  requestStream.write(mediaRequest);
+  const writer = requestStream.writable.getWriter();
+  await writer.ready();
```

Comment:
I don't think this is needed. If it is, then it should be `await writer.ready` (property not a method).

Reply:
Removed